boards/samr34-xpro/include/board.h

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (C) 2019 Mesotic SAS
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    boards_samr34-xpro Microchip SAM R34 Xplained Pro
+ * @ingroup     boards
+ * @brief       Support for the Microchip SAM R34 Xplained Pro board.
+ * @{
+ *
+ * @file
+ * @brief       Board specific definitions for the Microchip SAM R34
+ *              Xplained Pro board.
+ *
+ * @author      Dylan Laduranty <dylan.laduranty@mesotic.com>
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "cpu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name Semtech SX1276 configuration
+ * @{
+ */
+#define SX127X_PARAM_SPI                    (SPI_DEV(0))
+#define SX127X_PARAM_SPI_NSS                GPIO_PIN(1, 31)       /* D10 */
+#define SX127X_PARAM_RESET                  GPIO_PIN(1, 15)       /* A0 */
+#define SX127X_PARAM_DIO0                   GPIO_PIN(1, 16)      /* D2 */
+#define SX127X_PARAM_DIO1                   GPIO_PIN(0, 11)       /* D3 */
+#define SX127X_PARAM_DIO2                   GPIO_PIN(0, 12)       /* D4 */
+#define SX127X_PARAM_DIO3                   GPIO_PIN(1, 17)       /* D5 */
+#define SX127X_PARAM_PASELECT               (SX127X_PA_RFO)
+
+#define SX127X_PARAMS                   {   .spi       = SX127X_PARAM_SPI,     \
```

Comment:
You don't need to redefine this one since other defines below will be use in `sx127x_params.h`.